pipeline/ephys.py

```diff
@@ -78,10 +78,21 @@ class ProbeInsertionQuality(dj.Manual):
     definition = """  # Indication of insertion quality (good/bad) - for various reasons: lack of unit, poor behavior, poor histology
     -> ProbeInsertion
     ---
-    insertion_quality: enum('good', 'bad')
-    comment: varchar(1000)  # comment/reason for the 'good'/'bad' label
+    insertion_quality='good': enum('good', 'bad')
+    drift_presence=0: bool
+    number_of_landmarks: int
+    alignment_confidence=1: bool
+    comment='': varchar(1000)  # comment/reason for the 'good'/'bad' label
     """
 
+    class GoodPeriod(dj.Part):
+        definition = """
+        -> master
+        good_period_start: float  # seconds with respect to the start of the session
```

Comment:
avoid using floats in primary keys due to difficulties of equality comparisons.

Reply:
updated to use decimal type